### ISSUE TITLE
Node 16 is deprecated

### ISSUE THREAD

**Issue by TWiStErRob** [cardinalby/schema-validator-action#9]
Heads up:
https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

```
Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: cardinalby/schema-validator-action@v3. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/.
```

**Comment by cardinalby** [comment]
Thanks for the remainder, just updated

**Comment by TWiStErRob** [comment]
Thanks for the quick fix!

**Comment by MPV** [comment]
It (2377bdc0a633d3dfe1896c974b259cad077986fe) doesn't seem released to me yet:
- https://github.com/cardinalby/schema-validator-action/compare/3.1.0...master

Could you please release a new version (`4.x.x` or `3.2.x`) with this fix? 

As I'm getting:
>Node.js 16 actions are deprecated. Please update the following actions to use Node.js 20: cardinalby/schema-validator-action@76c68bfc941bd2dc82859f2528984999d1df36a4. For more information see: https://github.blog/changelog/2023-09-22-github-actions-transitioning-from-node-16-to-node-20/

When using it like this:
```yaml
      - uses: cardinalby/schema-validator-action@76c68bfc941bd2dc82859f2528984999d1df36a4 # 3.1.0
```

**Comment by cardinalby** [comment]
The fix is available under `@master`, `@v3` and `@3.1.1`
